app/src/main/res/values/strings.xml

```diff
@@ -1,40 +1,39 @@
 <resources>
     <string name="app_name">H3LP</string>
-    <string name="tos_text"><![CDATA[
-    <br/>All work and no play makes Jack a dull boy<br/>
-    <br/>All work and no play makes Jack a dull boy<br/>
-    <br/>All work and no play makes Jack a dull boy<br/>
-    <br/>All work and no play makes Jack a dull boy<br/>
-    <br/>All work and no play makes Jack a dull boy<br/>
-    <br/>All work and no play makes Jack a dull boy<br/>
-    <br/>All work and no play makes Jack a dull boy<br/>
-    <br/>All work and no play makes Jack a dull boy<br/>
-    <br/>All work and no play makes Jack a dull boy<br/>
-    <br/>All work and no play makes Jack a dull boy<br/>
-    <br/>All work and no play makes Jack a dull boy<br/>
-    <br/>All work and no play makes Jack a dull boy<br/>
-    <br/>All work and no play makes Jack a dull boy<br/>
-    <br/>All work and no play makes Jack a dull boy<br/>
-    <br/>All work and no play makes Jack a dull boy<br/>
-    <br/>All work and no play makes Jack a dull boy<br/>
-    <br/>All work and no play makes Jack a dull boy<br/>
-    <br/>All work and no play makes Jack a dull boy<br/>
-    <br/>All work and no play makes Jack a dull boy<br/>
-    <br/>All work and no play makes Jack a dull boy<br/>
-    <br/>All work and no play makes Jack a dull boy<br/>
-    <br/>All work and no play makes Jack a dull boy<br/>
-    <br/>All work and no play makes Jack a dull boy<br/>
-    <br/>All work and no play makes Jack a dull boy<br/>
-    <br/>All work and no play makes Jack a dull boy<br/>
-    <br/>All work and no play makes Jack a dull boy<br/>
-    <br/>All work and no play makes Jack a dull boy<br/>
-    <br/>All work and no play makes Jack a dull boy<br/>
-    <br/>All work and no play makes Jack a dull boy<br/>
-    <br/>All work and no play makes Jack a dull boy<br/>
-    <br/>All work and no play makes Jack a dull boy<br/>
-    <br/>All work and no play makes Jack a dull boy<br/>
-    <br/>All work and no play makes Jack a dull boy<br/>
-    ]]></string>
+    <string name="tos_text">
+        Acknowledgement
+
+        These are the Terms of Service governing the use of this Service and the agreement that operates between
```

Comment:
```suggestion
        These are the Terms of Service governing the use of this service and the agreement that operates between
```

Or is it a capital letter because it is a term defined properly ? I know the terms of services usually start with a definition of the terms that are used to describe the service (etc.), I'm not sure if this should be the case here